.gitlab/docker_common/publish_job_templates.yml

```diff
@@ -16,3 +16,5 @@
     - IMG_VARIABLES="$(sed -E "s#(${SRC_AGENT}|${SRC_DSD}|${SRC_DCA})#\1${ECR_RELEASE_SUFFIX}#g" <<<"$IMG_VARIABLES")"
     - IMG_SOURCES="$(sed -E "s#(${SRC_AGENT}|${SRC_DSD}|${SRC_DCA})#\1${ECR_RELEASE_SUFFIX}#g" <<<"$IMG_SOURCES")"
     - inv pipeline.trigger-child-pipeline --project-name "DataDog/public-images" --git-ref "main" --variables "IMG_VARIABLES,IMG_REGISTRIES,IMG_SOURCES,IMG_DESTINATIONS"
+    # Print a line showing the images that were pushed, for debug & use. Each image from IMG_DESTINATIONS needs to be prefixed with datadog/ to be pulled.
+    - echo "Published image(s) datadog/${IMG_DESTINATIONS//,/, datadog\/}"
```

Comment:
Should it be the job of `public-images`? (we already have it in the logs, but not everything in a single line)

Reply:
Yes, I think it would be nice to have in the output of the public-images job. If we do, then another nice to have would be to have a message here saying "go to this public-images job's log to see the list of published images", but I'm not sure that this would be easy to do with the `pipeline.trigger-child-pipeline` task (which should be as generic as possible).